src/components/CounterTabs/CounterTabs.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import CounterIcon from '../CounterIcon';
 
 
-const CounterTabs = ({tabClick, heroID, currentTab}) => {
-    console.log(`HERO: ${heroID}\n CURRENT" ${currentTab}\n`);
+const CounterTabs = ({tabClick, heroID, currentTab, counter}) => {
+
     return (
             <div onClick={() => tabClick(heroID)} className="tab col-1-of-3">
                     <div className={currentTab === heroID ? "row tab-border tab-active" : 
                     "row tab-border"}>
-                        <div className="col-1-of-2 tab-height tab-padding">
-                            <CounterIcon size="150" />
-                        </div>
+                    <div className="col-1-of-2 tab-height tab-padding">
+                    {counter.icon_url && <CounterIcon counterIcon={counter.icon_url} />}
```

Comment:
So if there's no `counter.icon_url`, this renders an empty `<div className="col-1-of-2 tab-height tab-padding"></div>`? Is that a desired effect?

Reply:
This Logic was something I did and is not needed. However, after reading your comment, even though we should always have an icon, would it be best to have a default picture in the event that the image fails to render?

Reply:
In general, as an engineer who creates things other people use, when I start a question with "Would it be best to have a default...?" I usually answer "Yes". 😄 

In this case, you might load the default as a placeholder, then update when you discover what the icon should be. Or you could set the icon to be what was passed in || the default.
```javascript
const heroIcon = counter.icon_url || default;
```